ui/packages/catalog-ui-search/src/main/webapp/extension-points/table-export/table-export.tsx

```diff
@@ -156,7 +161,7 @@ export const getDownloadBody = (downloadInfo: DownloadInfo) => {
     (property: string) =>
       filteredAttributes.includes(property) && !properties.isHidden(property)
   )
-  const cql = getCqlForSize(exportSize, selectionInterface)
+  const cql = getCqlForSize(exportSize, customExportCount, selectionInterface)
   const srcs = getSrcs(selectionInterface)
   const sorts = getSorts(selectionInterface)
   const count = Math.min(
```

Comment:
would it make more sense to get this count first then pass it into `getCqlForSize()`?

Reply:
makes sense